### ISSUE TITLE
teensy4-apa102-parallel?

### ISSUE THREAD

**Issue by GvBon** [FastLED/FastLED#1098]
Is it possible to output multiple apa102 strips (SPI) using parallel output on teensy4? 
I am aiming for 14 strips in parallel. 

something likethis that maybe has a shared clockpin?:
FastLED.addLeds<NUM_STRIPS, APA102, firstdatapin, clockpin, RGB>(leds, NUM_LEDS_PER_STRIP);

**Comment by cschill2020** [comment]
Same question:)

**Comment by zackees** [comment]
Yeah I've been thinking the same thing. Could be done with an interrupt. Multiple data lines could be set in one call. Then the clock signal strobed.

Simple in theory, some complexities in implementation. Nothing too major.

**Comment by zackees** [comment]
Reopening: this issue was closed automatically by a stale-bot run, not by a maintainer decision. That auto-close behavior was a mistake and has been disabled — FastLED issues will no longer be closed for inactivity. Apologies for the noise.
